README.md

```diff
@@ -29,8 +29,8 @@ Please move specific paths for projects instructions.
 The project is originally authored by
 [@wendux](https://github.com/wendux)
 with the organization
-[@flutterchina](https://github.com/flutterchina),
-started to maintained by
+[@flutterchina](https://github.com/flutterchina) - 
```

Comment:
Why should we use the dash here?

Reply:
The dash no specific reason, but the important part was the text after the dash or comma there. Just corrected the grammar.

Reply:
I would prefer:

```markdown
The project is originally authored by
[@wendux](https://github.com/wendux)
with the organization
[@flutterchina](https://github.com/flutterchina),
started being maintained by
[@cfug](https://github.com/cfug) since 2023.
```

Reply:
Yeah that's cool. Now fixed with [this commit](https://github.com/iamngoni/dio/commit/ad41f8940766947fe82592e66532bb6f02c52cea)